MovieApp/PresentationLayer/HomeScreen/HomeScreenViewController.swift

```diff
@@ -2,14 +2,39 @@ import UIKit
 
 class HomeScreenViewController: UIViewController {
 
+    let movies = MockHomeScreenData.data
+
     var navigationView: NavBarView!
     var searchBar: SearchBar!
-    var optionView: OptionBar!
+    var categoryCollection: CategoryCollectionView!
 
     override func viewDidLoad() {
         super.viewDidLoad()
 
         buildViews()
+        categoryCollection.setData(movies: movies)
+    }
+
+}
+
+// MARK: UICollectionViewDelegateFlowLayout
+
+extension HomeScreenViewController: UICollectionViewDelegateFlowLayout {
```

Comment:
Nope, CategoryCollectionView should extend UICollectionViewDelegateFlowLayout and define this functions.